app/Http/Controllers/EntriesController.php

```diff
@@ -79,16 +79,50 @@ public function add(Request $request)
 
     public function push(Request $request)
     {
+
         $validatedData = $request->validate([
-            'id' => 'required|exists:entries',
+            'id' => 'required|array',
```

Comment:
This is probably a better:

```
// Must be included for this to work
use Illuminate\Validation\Rule;

// ...

'id.*' => [
    'required',
    'integer',
    Rule::exists('entries', 'id'),
],
```

Reply:
Why is it better? Genuinely asking.

Reply:
Because, reasons.

On a more serious note, the current validation is only checking if the value passed is an array.

We can force that by using `id.*` instead of `id`, leaving us to apply validation per item, which in this case we want to make sure is passed (`required`), is a valid integer number and it exists in the `entries` table.